presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorMetadata.java

```diff
@@ -345,7 +350,9 @@ private ConnectorTableLayout getTableLayout(ConnectorSession session, RaptorTabl
     {
         ImmutableMap.Builder<String, RaptorColumnHandle> map = ImmutableMap.builder();
         long columnId = 1;
+        StorageTypeConverter storageTypeConverter = new StorageTypeConverter(typeManager);
```

Comment:
FMHO, maybe we can extract a small private function which check if a column's type is valid.